### PR TITLE
Add SQL parser test case for `ALTER INSTANCE instanceId SET XA_RECOVERY_NODES = value1,value2.`

### LINKED ISSUE #16357: [OpenForce 2022] Add SQL parser test case for `ALTER INSTANCE instanceId SET XA_RECOVERY_NODES = value1,value2`.
Hi community,

This issue is for https://github.com/apache/shardingsphere/discussions/12683#discussioncomment-2139441

### Background
`ALTER INSTANCE instanceId SET XA_RECOVERY_NODES  = value` is a syntax in DistSQL RAL, when ShardingSphereSQLParserEngine receives this SQL text, it parses it as AlterInstanceStatement. 

Currently we have enhanced the syntax from `ALTER INSTANCE instanceId SET XA_RECOVERY_NODES = value` to `ALTER INSTANCE instanceId SET XA_RECOVERY_NODES = value1,value2`. Therefore, it is necessary to **add a parsing test that supports multiple values**.

### Aim
Add SQL parser test case for  `ALTER INSTANCE instanceId SET XA_RECOVERY_NODES = value1,value2`.

- Affected files:
   - shardingsphere-test/shardingsphere-parser-test/src/main/resources/case/ral/common.xml(modify)
   - shardingsphere-test/shardingsphere-parser-test/src/main/resources/sql/supported/ral/common.xml(modify)

After all, execute DistSQLParserParameterizedTest#assertDistSQL() for validation

### Basic Qualifications
Java
Maven
### Example FYI
 - shardingsphere-test/shardingsphere-parser-test/src/main/resources/case/ral/common.xml:104 `(alter-instance sql-case-id="alter-instance-xa-recovery-nodes")`
 - shardingsphere-test/shardingsphere-parser-test/src/main/resources/sql/supported/ral/common.xml:55 `(distsql-case id="alter-instance-xa-recovery-nodes)`

search for "alter-instance-xa-recovery-nodes"

### PR DESCRIPTION
**Pull request by Amulya-coder**
Fixes  #16357.

Changes proposed in this pull request:
- Update line 104  `alter-instance sql-case-id="alter-instance-xa-recovery-nodes"`
- Update line 55 `distsql-case id="alter-instance-xa-recovery-nodes`

### CODE REVIEW

**Comment by lanchengx** [comment]
@Amulya-coder Hi, thanks for your submission.
Because the syntax has been enhanced, we should add a test case instead of modifying it.

File: shardingsphere-test/shardingsphere-parser-test/src/main/resources/sql/supported/ral/common.xml [hunk 1]
```diff
@@ -52,6 +52,6 @@
     <distsql-case id="prepare-distsql" value="PREPARE DISTSQL" />
     <distsql-case id="apply-distsql" value="APPLY DISTSQL" />
     <distsql-case id="discard-distsql" value="DISCARD DISTSQL" />
-    <distsql-case id="alter-instance-xa-recovery-nodes" value="ALTER INSTANCE 1.2.3.4@3307 SET XA_RECOVERY_NODES=1.2.3.4@3308" />
+    <distsql-case id="alter-instance-xa-recovery-nodes" value="ALTER INSTANCE 1.2.3.4@3307 SET XA_RECOVERY_NODES=1.2.3.4@3308,1.2.3.4@3307" />
```
[the remark this answers is not in the document]
Comment (Amulya-coder) [reply shardingsphere-test/shardingsphere-parser-test/src/main/resources/sql/supported/ral/common.xml:55]: @lanchengx could you guide me on how can I add a new test case here

[the remark this answers is not in the document]
Comment (lanchengx) [reply shardingsphere-test/shardingsphere-parser-test/src/main/resources/sql/supported/ral/common.xml:55]: @Amulya-coder  Keep the original test and add a new line below it as a new test case. 
The content is ` <distsql-case id="alter-instance-xa-recovery-nodes-with-multiple-values" value="ALTER INSTANCE 1.2.3.4@3307 SET XA_RECOVERY_NODES=1.2.3.4@3308,1.2.3.4@3307 " />`

In addition, you need to add the expected result in `shardingsphere-test/shardingsphere-parser-test/src/main/resources/case/ral/common.xml`

**Comment by Amulya-coder** [comment]
Hi @lanchengx, I have updated the pr but one check is failing could you please tell me why it's failing

**Review by lanchengx — APPROVED** [review]

**Pull request merged**